packages/docs/src/mobile/TextField/TextField.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { TextField, Typography, Provider, themes } from '@naturacosmeticos/natds-rn';
+import { withTheme } from '../../../addons/theme/provider';
+
+# Typography
```

Comment:
Why it says `Typography` here and in the preview below?

Reply:
This `Typography` was a mistake. The other ones was to use the Typography component in the documentation. They are okay. This was fixed in fb84473ab03a962c2ab984c2b3a7df80b29f5137